src/testdrive/src/action.rs

```diff
@@ -339,6 +339,9 @@ impl State {
         }
 
         // Grant initial privileges.
+        inner_client
```

Comment:
This is only going to be run for tests using testdrive, and only for those that do not use `--no-reset`. So e.g. zippy runs `CREATE CLUSTER` even before any testdrive involvement and this fails. I think testdrive should not be messing up with those settings, the `--bootstrap-role` functionality of environmentd should be used to provide those privileges for all tests. This is what the `MZ_` prefix diff in Item 1 will do.